Branch-SDK-Automation-TestBed/src/main/AndroidManifest.xml

```diff
@@ -16,69 +17,44 @@
             android:name=".activities.ShowQRCodeActivity"
             android:exported="true" />
         <activity
-            android:name=".activities.LogDataActivity"
-            android:exported="true"
-            android:screenOrientation="portrait" />
-        <activity
-            android:name=".activities.TrackContentActivity"
-            android:exported="true"
-            android:screenOrientation="portrait" />
-        <activity
-            android:name=".activities.MetadataActivity"
-            android:exported="true"
-            android:screenOrientation="portrait" />
-        <activity
-            android:name=".activities.UrlPreviewActivity"
+            android:name="io.branch.branchandroiddemo.activities.LogDataActivity"
             android:exported="true"
             android:screenOrientation="portrait" />
 
         <meta-data
             android:name="io.branch.sdk.BranchKey"
-            android:value="key_live_hcnegAumkH7Kv18M8AOHhfgiohpXq5tB" /> <!-- For your test app, if you have one; Again, use your actual test Branch key -->
+            android:value="key_live_fBcbpexoE7Nu1ONlzfl1BilgsFcsZT8T" /> <!-- For your test app, if you have one; Again, use your actual test Branch key -->
         <meta-data
             android:name="io.branch.sdk.BranchKey.test"
-            android:value="key_test_om2EWe1WBeBYmpz9Z1mdpopouDmoN72T" />
+            android:value="key_test_hdcBLUy1xZ1JD0tKg7qrLcgirFmPPVJc" />
```

Comment:
I noticed that this test key (`key_test_hdcBLUy1xZ1JD0tKg7qrLcgirFmPPVJc`) is paired with prod `key_live_hcnegAumkH7Kv18M8AOHhfgiohpXq5tB`, but `key_live_fBcbpexoE7Nu1ONlzfl1BilgsFcsZT8T ` is used.

Is there a test key that matches to the prod key for the branch app? Or is this intentional

Reply:
Fixed.